test/common/storagelib.py

```diff
@@ -80,16 +81,15 @@ def add_loopback_disk(self, size=50, name=None):
         # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1969408
         # It would be nicer to remove $F immediately after the call to
         # losetup, but that will break some versions of lvm2.
-        dev = self.machine.execute("F=$(mktemp /var/tmp/loop.XXXX); "
-                                   "truncate --size=%sMB $F; "
-                                   "losetup --show %s $F" % (size, name if name else "--find")).strip()
+        temp_path = f"/var/tmp/loop.{time.time()}"
```

Comment:
Please keep `mktemp`, nothing wrong with it.

Reply:
Ah, I though that your comment before mean to try to use another apporach than `F=`, okay